src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj

```diff
@@ -14,19 +14,19 @@
     <Description>eShopWorld common test stack</Description>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageProjectUrl>https://github.com/eShopWorld/tests-core</PackageProjectUrl>
-    <Copyright>Copyright eShopWorld 2019</Copyright>
+    <Copyright>Copyright eShopWorld 2020</Copyright>
     <PackageTags>Test, Unit Test, xUnit, Moq, FluentAssertions</PackageTags>
     <AssemblyName>Eshopworld.Tests.Core</AssemblyName>
     <RootNamespace>Eshopworld.Tests.Core</RootNamespace>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PackageReleaseNotes>added Level3 tests</PackageReleaseNotes>
+    <PackageReleaseNotes>Upgraded to netstandard2.1</PackageReleaseNotes>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Eshopworld.Tests.Core.Analyzers" Version="1.0.5" />
```

Comment:
Looks like we will need to update this in two steps, analyzers being a seperate one